okio/src/jvmMain/kotlin/okio/CipherSink.kt

```diff
@@ -127,13 +121,3 @@ private class CipherSink(
     return thrown
   }
 }
-
-/**
- * Returns a [Sink] that processes data using this [Cipher] while writing to
- * [sink].
- *
- * @throws IllegalArgumentException
- *  If this isn't a block cipher.
- */
-fun Cipher.sink(sink: BufferedSink): Sink =
```

Comment:
This API seems backwards anyway. You don't write `UTF_8.toBytes("hello")` you write `"hello".toBytes(UTF_8)`.